### PR TITLE
Blog post: preprint available

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
@pablopareja @rtobes @epareja please review if you want

File: source/_posts/2015-03-19-bio4j-preprint-available.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- news
+- preprint
+- publication
+---
+
+A citable preprint in the [bioRxiv][bioRxiv] describing Bio4j went online yesterday:
+
+- **[Bio4j: a high-performance cloud-enabled graph-based data platform][Bio4j preprint]**
+
+We hope will serve as a good introduction to what is Bio4j, and what it has to offer; especially so if, for getting a general idea of Bio4j, you would rather read prose than code. If you are using Bio4j for something that you want to publish, citing it is much easier now: all bioRxiv preprints are assigned a DOI. Comments, thoughts, opinions are all more than welcome! We will submit a paper based on this preprint to an open access journal. For completeness, here's the citation info and the abstract:
```
Comment (laughedelic) [inline source/_posts/2015-03-19-bio4j-preprint-available.markdown]: "We hope **it** will serve", no?

Comment (eparejatobes) [inline source/_posts/2015-03-19-bio4j-preprint-available.markdown]: it does. Thanks!

**Comment by laughedelic** [comment]
@bio4j/members congratulations! :tada: :+1: :smile:

**Comment by eparejatobes** [comment]
generating site and closing

**Comment by eparejatobes** [comment]
thanks @laughedelic!

**Pull request merged**